src/zfm20/zfm20.cxx

```diff
@@ -231,12 +130,12 @@ uint32_t ZFM20::getMillis()
   gettimeofday(&now, NULL);
 
```

Comment:
I'll bet this timing code could be replaced by a upm_clock_t and it's functions.  Not sure if you want to do that now though.